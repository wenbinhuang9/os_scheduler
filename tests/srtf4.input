101 50 1
10 10 42