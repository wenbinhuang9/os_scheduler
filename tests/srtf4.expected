10 53 1 43
101 51 0 1
1 22